kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeSideImportTab.js

```diff
@@ -182,76 +188,104 @@ define (
-                                        keys.push(key);
-                                    }
-                                    keys.sort(function(a,b) {return self.types[a]["name"].localeCompare(self.types[b]["name"])});
-                                    for (var keyPos in keys) {
-                                        addItem(keys[keyPos]);
-                                    }
-                                    $dropdown.select2({
-                                        minimumResultsForSearch: -1,
-                                        formatSelection: function(object, container) {
-                                            var display = '<span class="kb-parameter-data-selection">'+object.text+'</span>';
-                                            return display;
-                                        }
-                                    });
-
-                                    function addItem(key) {
-                                        var name = self.types[key]["name"];
-                                        $dropdown.append($('<option value="'+key+'">').append(name));
-                                    }
-                                }, this),
-                                $.proxy(function(error) {
-                                    self.showError(error);
-                                }, this)
-                            );
-                        }, this),
-                        $.proxy(function(error) {
-                            self.showError(error);
-                        }, this)
-                    );
+                    var keys = [];
+                    for (var key in self.types) {
+                        keys.push(key);
+                    }
+                    keys.sort(function(a,b) {return self.types[a]["name"].localeCompare(self.types[b]["name"])});
+                    for (var keyPos in keys) {
+                        addItem(keys[keyPos]);
+                    }
+                    $dropdown.select2({
+                        minimumResultsForSearch: -1,
+                        formatSelection: function(object, container) {
+                            var display = '<span class="kb-parameter-data-selection">'+object.text+'</span>';
+                            return display;
+                        }
+                    });
+
+                    function addItem(key) {
+                        var name = self.types[key]["name"];
+                        $dropdown.append($('<option value="'+key+'">').append(name));
+                    }
                 }, this),
                 $.proxy(function(error) {
                     self.showError(error);
                 }, this)
             );
             return this;
         },
-
-        showWidget: function(type) {
+        
+        getMethodSpecs: function(callback, errorCallback) {
+            var self = this;
+            var tag = Jupyter.narrative.sidePanel.$methodsWidget.currentTag;
```

Comment:
Might want to make the tag default to 'release' if that's not there. I think there's a small chance it might not initialize before this panel needs it.